### ISSUE TITLE
error to play new version of mcpe 1.16.210.05

### ISSUE THREAD

**Issue by rickbmsd** [minecraft-linux/mcpelauncher-manifest#473]
Error: Your previously downloaded Minecraft Version might be unsupported or just corrupted.
if you wanted to play a Beta or a new Release please wait patiently for an update,
please choose a compatible version from the profile Editor
otherwise if you have updated the Launcher recently.
e.g. a crash please delete it in Settings,
then download it again via the updated Launcher.

**Comment by rickbmsd** [comment]
(86x)

**Comment by GabeCoder** [comment]
Yea, I have the exact same problem.

**Comment by browe07** [comment]
Just wanted to chime in I have the same problem immediately as of 1.16.210.05.  Had the problem on an existing install on Manjaro and the same thing on a fresh install Lubuntu.  Was upgrading to follow a server upgrade I just did.  I'm a noob so that's about the best I can give as far as what might be relevant.  Want to take the opportunity to give thanks, while i'm here, for the work you've done.  I just switched to Linux a few months ago and your solution has helped a lot.

**Comment by GabeCoder** [comment]
> Just wanted to chime in I have the same problem immediately as of 1.16.210.05. Had the problem on an existing install on Manjaro and the same thing on a fresh install Lubuntu. Was upgrading to follow a server upgrade I just did. I'm a noob so that's about the best I can give as far as what might be relevant. Want to take the opportunity to give thanks, while i'm here, for the work you've done. I just switched to Linux a few months ago and your solution has helped a lot.

As it seems, the problem is either the launcher or copy, as no matter the software or hardware, the problem persists.

**Comment by edam** [comment]
Duplicate of #470

**Comment by Gamer1039** [comment]
Yes expierenced the same issue which prevents me to play with friends
